stories/Molecules/Select/select.scss

```diff
@@ -0,0 +1,34 @@
+@import '../../assets/scss/variables';
+@import '../../assets/scss/mixins';
+@import '../../assets/scss/breakpoints';
+
+.select-default {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background: $color-white url('../../assets/images/Icon/Chevron-down.svg') no-repeat right 22px center;
+  border: 2px solid $color-black;
+  border-radius: 0;
+  color: #000;
+  cursor: pointer;
+  display: flex;
+  display: block;
+  flex-direction: column;
+  font-family: $Font-Family-ProximaSemiBold;
+  font-size: $font-size-16;
+  font-weight: 400;
+  max-width: 100%;
+  min-width: 300px;
+  padding: 16px 50px 16px 16px;
+  position: relative;
+  text-align: left;
+  text-transform: uppercase;
+}
+
+[dir='rtl'] {
+  .select-default {
+    background-position: left 22px center;
+    padding: 16px 16px 16px 50px;
+    text-align: right;
+  }
+}
```

Comment:
@Ashish-srij Please use the proper variables and create the mixins

Reply:
Done